xclim/indices.py

```diff
@@ -443,34 +588,74 @@ def tg_mean(tas, freq='YS'):
 
 # @valid_daily_min_temperature
 def tn10p(tasmin, p10, freq='YS'):
-    """Days with daily minimum temperature below the 10th percentile of the reference period."""
+    """Days with daily minimum temperature below the 10th percentile of the reference period.
+
+    Parameters
+    ----------
+    tasmin : xarray.DataArray
+      Minimum daily temperature [℃] or [K]
+    p10 : float
+    freq : str, optional
+      Resampling frequency
+
+    """
     return (tasmin.groupby('time.dayofyear') < p10).resample(time=freq).sum(dim='time')
 
 
 @valid_daily_min_temperature
 def tn_max(tasmin, freq='YS'):
-    """Maximum of daily minimum temperature."""
+    """Highest minimum temperature
+
+    Maximum of daily minimum temperature.
+
+    Parameters
+    ----------
+    tasmin : xarray.DataArray
+      Minimum daily temperature [℃] or [K]
+    freq : str, optional
+      Resampling frequency
+    """
     return tasmin.resample(time=freq).max(dim='time')
 
 
 @valid_daily_min_temperature
 def tn_mean(tasmin, freq='YS'):
-    """Mean of daily minimum temperature."""
+    """Mean minimum temperature
+
+    Mean of daily minimum temperature.
+
+    Parameters
+    ----------
+    tasmin : xarray.DataArray
+      Minimum daily temperature [℃] or [K]
+    freq : str, optional
+      Resampling frequency
+    """
     arr = tasmin.resample(time=freq) if freq else tasmin
     return arr.mean(dim='time')
 
 
 @valid_daily_min_temperature
 def tn_min(tasmin, freq='YS'):
-    """Minimum of daily minimum temperature."""
+    """Extreme minimum temperature
```

Comment:
Here you're using extreme, whereas you used Highest earlier. I suggest we stick with the ICCLIM definitions whenever reasonable.

Reply:
Sounds good.